Jellyfin.Networking/Configuration/NetworkConfiguration.cs

```diff
@@ -0,0 +1,223 @@
+#pragma warning disable CA1819 // Properties should not return arrays
+
+using System;
+using MediaBrowser.Model.Configuration;
+
+namespace Jellyfin.Networking.Configuration
+{
+    /// <summary>
+    /// Defines the <see cref="NetworkConfiguration" />.
+    /// </summary>
+    public class NetworkConfiguration
+    {
+        private string _baseUrl = string.Empty;
```

Comment:
Is BaseUrl now set in multiple areas?

Reply:
This was a change today in light of @cvium comment above about the number of config options.

As a response, I've copied all network settings into the NetworkConfiguration file - and in PR2 is a small bit of migration code, that copies the values from system.xml into network.xml, and handles the web interface integration.

The values used are those specified in the web code as being on the network page.

The values are still in ServerConfiguration for backwards compatibility and until the web-interface code is modified.

The system checks for network.xml and uses the values in there. If it's missing, the values in ServerConfiguration are used to create one.

If network.xml exists, values manually entered into system.xml are ignored.